k8s/iabweb/kustomization.yaml

```diff
@@ -1,7 +1,6 @@
 namespace: iabwww
-namePrefix: iabwww-
```

Comment:
Using `namePrefix` has been confusing for me at times (since it's not super obvious what each identifier will be called in the resulting manifest). However, we should note that it's still being used in other apps - just some inconsistency to be aware of in the meantime.